main/options.c

```diff
@@ -247,7 +247,7 @@ static optionDescription LongOptionDescription [] = {
  {1,"      Include selected extension fields (flags: \"afmikKlnsStzZ\") [fks]."},
  {1,"  --fields-<LANG|*>=[+|-]flags"},
  {1,"      Include selected <LANG> own extension fields"},
- {1,"      (flags: --list-flags=<LANG>)."},
+ {1,"      (fields: --list-fields=<LANG>)."},
```

Comment:
Thank you.

I think `flags: --list-fields=<LANG>` is better. Here `flags` means the name of parameter of `--fields-` option.